OpenShift/99_post_install.sh

```diff
@@ -32,6 +33,12 @@ create_bridge(){
   fi
 }
 
+extra_registry(){
+  if [ "${EXTRA_REGISTRY}" != "" ] ; then
+    envsubst < ${EXTRA_REGISTRY_FILE}.template > ${EXTRA_REGISTRY_FILE}
+  fi
+}
+
 apply_mc(){
```

Comment:
I think we also need to add oc create -f ${POSTINSTALL_ASSETS_DIR}/99-extra-registry.yaml since the existing oc create in the apply_mc function only matches yaml file names including master or worker.

Reply:
good catch!

Reply:
Instead, different mc are created for masters and workers